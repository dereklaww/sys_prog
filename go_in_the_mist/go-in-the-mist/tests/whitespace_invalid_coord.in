 who
who

    
who
who A0
who   who
place A00
place 0A
place B1
place 00000A
place A2  place B2
place B2
resign