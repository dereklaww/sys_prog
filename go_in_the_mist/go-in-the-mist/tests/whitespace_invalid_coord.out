Invalid!
B
Invalid!
Invalid!
Invalid!
Invalid!
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid!
White wins!
B1B2
Thank you for playing!